modules/check_in/config/routes.rb

```diff
@@ -18,6 +18,8 @@
   namespace :v2, defaults: { format: :json } do
     resources :patient_check_ins, only: %i[show create]
     resources :sessions, only: %i[show create]
+    get 'sessions/:id/appointments', to: 'appointments#index'
```

Comment:
not significant, but I'm debating this is a subresource of sessions, or a stand-alone resource. we can circle back on this when we pick it back up